docs/pipelines.md

```diff
@@ -39,13 +39,14 @@ following fields:
     [`PipelineResources`](resources.md) of which types the `Pipeline` will be
     using in its [Tasks](#pipeline-tasks)
   - `tasks`
-    - `resources.inputs` / `resource.outputs`
-      - [`from`](#from) - Used when the content of the
-        [`PipelineResource`](resources.md) should come from the
-        [output](tasks.md#outputs) of a previous [Pipeline Task](#pipeline-tasks)
+      - `resources.inputs` / `resource.outputs`
+          - [`from`](#from) - Used when the content of the
+            [`PipelineResource`](resources.md) should come from the
+            [output](tasks.md#outputs) of a previous [Pipeline Task](#pipeline-tasks)
       - [`runAfter`](#runAfter) - Used when the [Pipeline Task](#pipeline-tasks)
         should be executed after another Pipeline Task, but there is no
         [output linking](#from) required
+      - [`description`](#description) - Description of the Pipeline.
```

Comment:
The lists are a bit confusing here but I believe description is an Optional top-level field of PipelineSpec, right?  I think we should put this field above [`resources`], just after line 37.

Reply:
@sbwsg Update the PR with your review comment. Thanks